packages/truffle-compile-vyper/index.js

```diff
@@ -112,15 +113,15 @@ function execVyper(options, source_path, callback) {
 }
 
 // compile all options.paths
-function compileAll(options, callback) {
+async function compileAll(options, callback) {
   options.logger = options.logger || console;
 
   compile.display(options.paths, options);
 
   async.map(
     options.paths,
     function(source_path, c) {
-      execVyper(options, source_path, function(err, compiled_contract) {
+      execVyper(options, source_path, async function(err, compiled_contract) {
```

Comment:
Not sure about this, but I'm pretty sure we can't make this `async` either. Can we use the `readFileSync` instead?

Reply:
Yeah I wasn’t a fan of making these async- it works in the sense that we get the file contents but I figured it was the wrong way.  

Was hesitant to use readFileSync because it’s synchronous but happy to change it if you think that’s the better way!

Reply:
Definitely! Otherwise who will `.then` the promise?

Reply:
Fair enough! Was concerned about having a blocking method in case someone has a lot of Vyper files or something but that's probably not that big a concern at this time? 

Made the change, thanks for the review!